dallinger/prolific.py

```diff
@@ -674,8 +674,13 @@ def prolific_service_from_config():  #
 
 
 def dev_prolific_service_from_config():
+    from dallinger.config import get_config
     from dallinger.prolific import DevProlificService
 
+    config = get_config()
+    config.load()
+    config.set("prolific_workspace", "62fce6fff0a78eb4f3ebc09c")
```

Comment:
I think that '62fce6fff0a78eb4f3ebc09c' should not be hardcoded but instead made a variable, this will help readability and maintainability

Reply:
Addressed